behind-the-veil-siteroot/imports/ui/components/navigationBar/NavigationBar.jsx

```diff
@@ -115,8 +152,11 @@ export const NavigationBar = () => {
                                  autoCloseMenu();
                              }}
                              className={baseUrl === UrlBasePath.MESSAGES ?
-                                 "main-text navbar-link-active lg:border-b-2 lg:border-dark-grey p-3" :
-                                 "main-text navbar-link-inactive p-3"}>Messages</NavLink>
+                                 "main-text navbar-link-active lg:border-b-2 lg:border-dark-grey p-3 relative" :
+                                 "main-text navbar-link-inactive p-3 relative"}>Messages {unreadMessages && (
+                                    <span className="absolute -top-0 -right-1 block h-2.5 w-2.5 rounded-full bg-red-600"></span>
```

Comment:
re: your message about making the red dot closer, have you tried something like replacing this line with:
```
<span className="absolute top-[6px] right-[6px] block h-2.5 w-2.5 rounded-full bg-red-600"></span>
```

to bring it closer. it just specifies the exact pixel width for the top and right stylings instead of using tailwind's predefined distances

Reply:
Thank you! I used that trick and it helped bring it a bit closer - I still can't get it as close as I'd like but it's better than before